src/sass/_storage.scss

```diff
@@ -83,3 +93,16 @@
     margin-top: -7px;
   }
 }
+
+.storage-pool-meter-container {
+  align-items: center;
+  display: flex;
+}
+
+.resource-link {
+  &.storage-pool-table {
+    justify-content: center;
+    min-width: 6rem;
+    width: 6rem;
+  }
+}
```

Comment:
Those chips should match the content width, the text should not be centred, but left aligned. The space that is reserved for them should be equal. Like there should be some space between the chip and the meter if a name is very short. And the Name chip should go all the way next to the meter if the name is long.

![image](https://github.com/user-attachments/assets/56fbfaa6-f1f6-4f6b-a058-d0cd60394eb4)

Reply:
In the horizontal axis the names of the chips should be aligned with the other columns, they are a bit too low. The meters seem to be okay, but the chip content should be aligned with the meter. I think if the chips are a bit higher, their text will align also in secondary lines within the cell with the meter.

![image](https://github.com/user-attachments/assets/185f7fee-77a8-4a54-9d9e-b86c28e34a0b)